### PR TITLE
Case list option set

### PR DESCRIPTION
**Pull request by snopoke**
http://manage.dimagi.com/default.asp?220461
Caused by a timing issue of when different JS scripts are run.

Solved by setting option directly instead of indirectly via JS. See https://github.com/dimagi/commcare-hq/pull/10929#issuecomment-200966576 for more details.

### CODE REVIEW

**Comment by snopoke** [comment]
@NoahCarnahan

**Comment by snopoke** [comment]
buddy @dannyroberts

File: corehq/apps/app_manager/templates/app_manager/partials/case_list_setting.html [hunk 1]
```diff
@@ -10,10 +10,9 @@
         ></span>
     </label>
     <div class="col-sm-4">
-        <select class="form-control" type="text" name="{{ SLUG }}-show"
-                data-value="{% if case_list.show %}true{% else %}false{% endif %}">
-            <option value="false">Don't Show</option>
-            <option value="true">Show</option>
+        <select class="form-control" type="text" name="{{ SLUG }}-show">
+            <option value="false" {% if not case_list.show %}selected{% endif %}>Don't Show</option>
```
Comment (czue) [inline corehq/apps/app_manager/templates/app_manager/partials/case_list_setting.html]: nit: trans

**Comment by czue** [comment]
:+1: on :white_check_mark:

**Pull request merged**